.travis.yml

```diff
@@ -44,7 +44,7 @@ script:
     - cd ../build-bl-f7
     - make -f ../Makefile OPT_GCC_ARM=$GCC_DIR ROOTLOC=".." BUILDFOR="F7" TRX_ID="ovi40" TRX_NAME="OVI40" bootloader
     - cd ../build-fw-f4-ili9486-480
-    - make -f ../Makefile OPT_GCC_ARM=$GCC_DIR ROOTLOC=".." LCD_TYPE=1 all
+    - make -f ../Makefile OPT_GCC_ARM=$GCC_DIR ROOTLOC=".." CONFIGFLAGS="-DEXTERNAL_USE_GFX_CONFIG -DUSE_DISP_480_320 -DUSE_GFX_ILI9486" all 
```

Comment:
Sorry for late comments, forgot to press finish review :)

Reply:
Hi Slawek,
how can this change disable your check? There is no active reference in the current code which relates to LCD_TYPE since you deleted the respective statements in uhsdr_board.h.

If you are using it for some purpose,  I was not aware of this. 
Easiest and safest solution: simply add the respective statement in your branch after rebasing. This will keep these changes in even in future rebase action.

I would still want to have a build with not all display drivers enabled as part of the travis. 

BTW, the headless build is just a by-product of our approach, not an explict goal. I just noticed that we have that compiling without a problem. But in fact using such a binary is not much different from one with the wrong display drivers compiled in, you'll get the signal that no display has been detected.

Reply:
hmm, not tested as yet of course, but defining the EXTERNAL_USE_GFX_CONFIG with only 480x320 driver you disabled the check for other drivers by undefining them in uhsdr_board.h line#79. Why do you need to disable display drivers check for travis?

Reply:
Hi Slawek,
this is only one of multiple builds we run inside the single travis job, have a look at the full .travis.yml 

We cannot test all of the different combinations of individual drivers and resolutions. 
Purpose of this build is to test a minimal configuration using a single driver plus a single resolution. Now that we are discussing, it occurs to me that the USE_DISP_480x320 may make it two resolutions. Anyway, that is the motivation. 
In corner cases we may have to provide for instance stripped down bootloaders with just a single display driver to keep the size it below the limit.

Reply:
Yes you're right, I overlooked the previous fw build with the f4, correct me but I think the case of exclusive fw build for f4+480x320 is a relict when we tested the drivers at early stage before unification (that's why i removed the LCDTYPE from defines). But for the whole idea of striping down the bootloaders out of not necessary drivers is understood.